ddtrace/tracer/tracer.go

```diff
@@ -265,6 +265,9 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		Start:        startTime,
 		taskEnd:      startExecutionTracerTask(operationName),
 		noDebugStack: t.config.noDebugStack,
+		Meta: map[string]string{
+			"_dd.hostname": t.config.hostname,
+		},
 	}
```

Comment:
We should probably add this after... It might be empty. If it's empty we shouldn't add it.
```suggestion
	}
	if host := t.config.hostname; host != "" {
		span.setMeta(keyHostname, host)
	}
```
We should also remove this setting on line 294 (no longer needed).

Reply:
Assigning a value to the `host` variable is more efficient?

Reply:
Not more efficient, just less to type. You don't have to type the whole `t.config.hostname` sausage twice 🙂 But you can do it that way too if you prefer. I don't mind.